provision/keycloak/keycloak-legacy-postgres.yml

```diff
@@ -0,0 +1,36 @@
+version: '2.2'
+
+volumes:
+  postgres_data:
+      driver: local
+
+services:
+  postgres:
+      image: postgres
+      container_name: postgres
+      cpuset: "0-1"
```

Comment:
I'd rather keep the cpuset information to a more full-blown setup based on Minikube and not docker-compose.
I saw that those settings broke my setup when running Docker-Compose with Podman.

Reply:
interesting, I am running docker-compose and podman, simultaneously and it didn't affect my environment. If you think its appropriate to make this part of a separate setup, I can pull this one out.

Reply:
After running

```
export DOCKER_HOST="unix:/run/user/$(id -u)/podman/podman.sock"
podman system service -t 3600 &
```

The error message I get: 

```
ERROR: for cf6112a55e50_keycloak_postgres_1  Cannot start service postgres: the requested cgroup controller `cpuset` is not available: OCI runtime error

ERROR: for postgres  Cannot start service postgres: the requested cgroup controller `cpuset` is not available: OCI runtime error
ERROR: Encountered errors while bringing up the project.
```

Therefore, please remove it.